common/locales/en-US/main.ftl

```diff
@@ -76,8 +76,9 @@ friends = Friends
     .incoming_requests = Incoming Requests
     .outgoing_requests = Outgoing Requests
     .cancel = Cancel Request
-    .requested = Requested 
-    .sent = Sent
+    .requested = Requested { $time }
```

Comment:
I wonder if we should remove the time out of this since in warp, it never shows the time when a user was blocked, and friend request no longer does (since I changed the corresponding functions to show DID rather than a struct of the request due to it being a bit redundant and causing more confusion).